src/tests/Equinor.ProCoSys.IPO.Query.Tests/GetInvitationById/GetInvitationByIdQueryHandlerTests.cs

```diff
@@ -0,0 +1,188 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Equinor.ProCoSys.IPO.Infrastructure;
+using Equinor.ProCoSys.IPO.Query.GetInvitationById;
+using Equinor.ProCoSys.IPO.Test.Common;
+using Equinor.ProCoSys.IPO.Test.Common.ExtensionMethods;
+using Fusion.Integration.Http.Models;
+using Fusion.Integration.Meeting;
+using Fusion.Integration.Meeting.Http.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using ServiceResult;
+
+namespace Equinor.ProCoSys.IPO.Query.Tests.GetInvitationById
+{
+    [TestClass]
+    public class GetInvitationByIdQueryHandlerTests : ReadOnlyTestsBase
+    {
+        private Invitation _invitation;
+        private int _invitationId = 246;
+        
+        private Mock<IFusionMeetingClient> _meetingClientMock;
+
+        protected override void SetupNewDatabase(DbContextOptions<IPOContext> dbContextOptions)
+        {
+            using (var context = new IPOContext(dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var MeetingId = new Guid("11111111-2222-2222-2222-333333333333");
+                var PersonAzureOid = new Guid("44444444-5555-5555-5555-666666666666");
+                const string ProjectName = "Project1";
+                const string Description = "Description";
+                const string CommPkgNo = "CommPkgNo";
+
+                var FunctionalRoleParticipant = new Participant(
+                    TestPlant,
+                    Organization.Contractor,
+                    IpoParticipantType.FunctionalRole,
+                    "FR1",
+                    null,
+                    null,
+                    "FR1@email.com",
+                    null,
+                    0);
+
+                var PersonParticipant = new Participant(
+                    TestPlant,
+                    Organization.Contractor,
```

Comment:
This doesnt affect your test, but in reality we should never have a participant with sortkey=1, with organization.Contractor. If sortkey=1 the organization is ConstructionCompany